### ISSUE TITLE
[Feature Request] el-select have option to open pop up up or below

### ISSUE THREAD

**Issue by thamibn** [element-plus/element-plus#2658]
### Existing Component
Yes

### Component Name
el-select

### Description
Can we have a feature to tell el-select to open up or below, or it can do it automatically by calculating the viewport width and decide if it wants to open up or below.

**Comment by JeremyWuuuuu** [comment]
Yes of course, I am current working on this, but I think it would take a little bit longer than I expected since we have other stuff getting in our way that we need to eliminate first, I will let you know once we have updated that.

**Comment by Dsaquel** [comment]
In newer versions you can use `placement` + `fallback-placements` to meet your needs.
